pg-index-health-model/src/main/java/io/github/mfvanek/pg/model/validation/Validators.java

```diff
@@ -81,4 +81,11 @@ public static <T> void validateThatNotEmpty(@Nonnull final List<T> columnsInCons
             throw new IllegalArgumentException("columnsInConstraint cannot be empty");
         }
     }
+
+    public static double validPercent(final double percentValue, @Nonnull final String argumentName) {
```

Comment:
We need to get rid of `AdditionalValidators` and code duplication

Reply:
I did not use validPercent of AdditionalValidators because it requires additional dependency. Should I fix it by adding a dependency?

Reply:
No, just leave it as it is.